core/src/main/kotlin/defaultConfiguration.kt

```diff
@@ -72,6 +72,10 @@ public data class SourceLinkDefinitionImpl(
     override val remoteLineSuffix: String?,
 ) : DokkaConfiguration.SourceLinkDefinition {
 
+    public constructor(
+        localDirectory: String, remoteUrl: String, remoteLineSuffix: String?
+    ) : this(localDirectory, URL(remoteUrl), remoteLineSuffix)
```

Comment:
Since [the `URL` constructor is deprecated](https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/net/URL.html#constructor-deprecation) as of Java 20, maybe use the recommended replacement `URI(remoteUrl).toURL()` instead? That has been available since at least Java 11.

Reply:
Resolved in 61e0f3e47c0b9d726a51528fafdceac46dfe8673. Thank you!